external/vulkancts/modules/vulkan/wsi/vktWsiSwapchainTests.cpp

```diff
@@ -2417,6 +2418,81 @@ tcu::TestStatus destroyNullHandleSwapchainTest (Context& context, Type wsiType)
 	return tcu::TestStatus::pass("Destroying a VK_NULL_HANDLE surface has no effect");
 }
 
+tcu::TestStatus acquireTooManyTest (Context& context, Type wsiType)
```

Comment:
Please have a look at the coding conventions framework/delibs/coding_guidelines/de-coding-guidelines.html

Reply:
Dunno, I only emulated the style of the test above. Except the bad indentation+aligning.
I gave the styleguide a quick scan, but haven't found anything relevant. It would be productive if you would point me to the specific rule.

Reply:
PS: though I should probably merge the code of the two tests.

Reply:
indentation+aligning should be fixed

Reply:
Sorry, forgot about this. What remains to be done here?
I see no problems with whitespace in my code. Tabs are used for indentation, spaces for aligning; any other way does not really work and gives tabs a bad rep.

Reply:
It's being reviewed by the IHVs. I'll have more info after tomorrow.